File: 1132.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let a = parseInt(lines.shift()); let b = parseInt(lines.shift());
let total = 0;
for(var i = Math.min(a,b); i < Math.max(a,b); i++){
    if(i % 13 !== 0){
        total+=i;}
}
console.log(total);

File: 1132.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a = input.nextInt(), b = input.nextInt(), total = 0;
        for(int i = Math.min(a,b); i < Math.max(a,b)+1; i++) {
                if(i % 13 != 0) {
                        total += i;
                }
        }
        System.out.println(total);
}
}
